utils/rpms/daos.spec

```diff
@@ -207,12 +207,11 @@ packages
 Summary: The DAOS test suite
 Requires: %{name}-client%{?_isa} = %{version}-%{release}
 Requires: %{name}-admin%{?_isa} = %{version}-%{release}
-Requires: python3-distro
-Requires: python3-tabulate
-Requires: python3-defusedxml
+Requires: %{name}-devel%{?_isa} = %{version}-%{release}
```

Comment:
Why does the `client-tests` package now need `daos-devel`?

Reply:
See the PR description and comments on the PR for detail.

pydaos is now installed into the python virtual environment via pip, and to be able to do that the daos headers need to be installed - hence the pydaos source is in the daos-devel package.